frontend/src/components/CreateEvent/CreateEvent.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react";
```

Comment:
nice but where is the functionality?
what happens when i click on the Submit button